From b1714b27758b44384c5ffcc8ae5dbc442468019c Mon Sep 17 00:00:00 2001
From: danielk43 <11856117+danielk43@users.noreply.github.com>
Date: Fri, 21 Jun 2024 18:26:59 -0400
Subject: [PATCH 2/2] lynx: Update apps

---
 Android.bp     | 22 ----------------------
 lynx-vendor.mk |  2 --
 2 files changed, 24 deletions(-)

diff --git a/Android.bp b/Android.bp
index 99c7c77..87aa7f3 100644
--- a/Android.bp
+++ b/Android.bp
@@ -5,28 +5,6 @@
 soong_namespace {
 }
 
-android_app_import {
-	name: "PixelCameraServicesL10",
-	owner: "google",
-	apk: "proprietary/product/app/PixelCameraServicesL10/PixelCameraServicesL10.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	product_specific: true,
-}
-
-android_app_import {
-	name: "VZWAPNLib",
-	owner: "google",
-	apk: "proprietary/product/app/VZWAPNLib/VZWAPNLib.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	product_specific: true,
-}
-
 android_app_import {
 	name: "CarrierSettings",
 	owner: "google",
diff --git a/lynx-vendor.mk b/lynx-vendor.mk
index 7d80ea5..0866eb1 100644
--- a/lynx-vendor.mk
+++ b/lynx-vendor.mk
@@ -66,8 +66,6 @@ PRODUCT_COPY_FILES += \
     vendor/google/lynx/proprietary/system_ext/lib64/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so \
 
 PRODUCT_PACKAGES += \
-    PixelCameraServicesL10 \
-    VZWAPNLib \
     CarrierSettings \
     WfcActivation \
     OemRilHookService \
-- 
2.39.2

